packages/react-cosmos/src/components/Loader.jsx

```diff
@@ -1,3 +1,5 @@
+/* global parent */
```

Comment:
Is this needed? ESLint doesn't render any errors for me when I remove it (because [this](https://github.com/react-cosmos/react-cosmos/blob/7a90876a11ea257a122c2d25295c482abef2e30c/packages/react-cosmos/src/.eslintrc.json#L3))